### ISSUE TITLE
Uncaught TypeError: Cannot read property 'positionedAtEndOfWrappingLine' of undefined

### ISSUE THREAD

**Issue by macosxguru** [FoldingText/foldingtext-for-atom#13]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 0.199.0
**System**: Mac OS X 10.10.3
**Thrown From**: [foldingtext-for-atom](https://github.com/FoldingText/foldingtext-for-atom) package, v0.1.0
### Stack Trace

Uncaught TypeError: Cannot read property 'positionedAtEndOfWrappingLine' of undefined

```
At /Users/mithoo/.atom/packages/foldingtext-for-atom/lib/editor/outline-editor-element.coffee:494

TypeError: Cannot read property 'positionedAtEndOfWrappingLine' of undefined
  at ft-outline-editor.module.exports.OutlineEditorElement.focus (/Users/mithoo/.atom/packages/foldingtext-for-atom/lib/editor/outline-editor-element.coffee:494:22)
  at atom-pane.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-element.js:61:18)
  at atom-pane.PaneElement.activated (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-element.js:125:19)
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at Pane.module.exports.Pane.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:612:27)
  at TreeView.module.exports.TreeView.unfocus (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tree-view/lib/tree-view.js:404:45)
  at TreeView.module.exports.TreeView.entryClicked (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tree-view/lib/tree-view.js:435:18)
  at tree-view-file.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tree-view/lib/tree-view.js:178:26)
  at space-pen-div.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at space-pen-div.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```
### Commands

```
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "language-c",
      "language-go",
      "language-make",
      "language-objective-c",
      "atom-color-highlight",
      "atom-color-highlight",
      "color"
    ],
    "themes": [
      "atom-light-ui",
      "atom-light-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
color-picker, v1.7.0
foldingtext-for-atom, v0.1.0
fountain, v0.1.1
recent-files, v0.3.0
webbox-color, v0.5.6

# Dev
No dev packages
```

**Comment by jessegrosjean** [comment]
Thanks for submitting this... do you remember what you were doing when it happened? I don't think I've seen this exception myself. Thanks!

**Comment by jessegrosjean** [comment]
Never mind I can reproduce it now. Looks to be a new issue due to some change in Atom 0.199.

**Comment by jessegrosjean** [comment]
This should be fixed now. The issue was that pre Atom 0.199 fs and path were in the global namespace and FoldingText was using those globals. But in Atom 0.199 they were removed from global namespace and that caused a bunch of crashes. It's fixed now in FoldingText 0.1.1, thanks for the quick report.
